FizzBuzzApp/Program.cs

```diff
@@ -1,3 +1,15 @@
 ﻿// See https://aka.ms/new-console-template for more information
-
-Console.WriteLine("Hello, World!");
\ No newline at end of file
+for (int i = 1; i < 101; i++) {

```

Comment:
Nitpick: Since the upper bound value we want to set is 100, its a bit less obvious that this is the case when using 101 in the code. Whilst its not much of a problem now, if in future you changed the function to allow the user to pick how far to go e.g. fizzbuzz(200) then you'd need to do some janky +1 to it. Changing this to <= means you can use the number the user gives without needing to change it :D